src/main/java/org/broadinstitute/hellbender/utils/iterators/IntervalOverlappingIterator.java

```
@@ -48,6 +48,8 @@ public IntervalOverlappingIterator(Iterator<T> iterator, List<SimpleInterval> in
         this.iterator = iterator;
         this.intervals = intervals.iterator();
         this.dictionary = dictionary;
+
+        //TODO: If the input intervals is empty, won't this throw an exception?  Is that planned?
```

Yes it would so please address.... try to fail early!